### PR TITLE
feat(unlock-js): Add grantKeys to walletService

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

This adds grantKeys function to wallet service.

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated the docs to reflect my changes if applicable
- [x] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/unlock-js/src/PublicLock/v4/index.js [hunk 1]
```diff
@@ -12,6 +13,7 @@ export default {
   purchaseKey,
   purchaseKeys,
   grantKey,
```
Comment (julien51) [inline packages/unlock-js/src/PublicLock/v4/index.js:15]: Let's make sure to refactor `grantKey` so it uses `grantKeys` to make things DRY :)

File: packages/unlock-js/tsconfig.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
         "skipLibCheck": true,
     },
     "include": [
-        "./src/**/*.ts"
+        "./src/**/*.ts",
+        "src/PublicLock/v4/grantKeys.js"
```
Comment (julien51) [inline packages/unlock-js/tsconfig.json:18]: Any reason for this?

**Review by julien51 — COMMENTED** [review]
Good start! Let's DRY things a bit please :)

File: packages/unlock-js/src/PublicLock/v6/grantKeys.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import { ETHERS_MAX_UINT } from '../../constants'
```
[the remark this answers is not in the document]
Comment (searchableguy) [reply packages/unlock-js/src/PublicLock/v6/grantKeys.js:1]: The grant key function in v6 and v4 are slightly different.

File: packages/unlock-js/tsconfig.json (hunk 2 above)
Comment (searchableguy) [reply packages/unlock-js/tsconfig.json:18]: Editor going rogue.

File: packages/unlock-js/src/PublicLock/v4/index.js (hunk 1 above)
Comment (searchableguy) [reply packages/unlock-js/src/PublicLock/v4/index.js:15]: done

**Review by julien51 — APPROVED** [review]
Please make sure you add the changes to `CHANGELOG` before merging!

**Pull request merged**